.claude/skills/prow-ci/analyze_failure.py

```
@@ -0,0 +1,269 @@
+import json
+import os
+import re
+import sys
+import xml.etree.ElementTree as ET
+from pathlib import Path
+
+
+def parse_junit_xml(xml_file):
+    """Parse JUnit XML and extract failures."""
+    try:
+        tree = ET.parse(xml_file)
```

Ruff (0.15.15)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)